cmds/mint.go

```diff
@@ -19,7 +19,7 @@ type MintCommand struct {
 	cmds.OperationFlags
 	Sender           cmds.AddressFlag    `arg:"" name:"sender" help:"sender address" required:"true"`
 	Contract         cmds.AddressFlag    `arg:"" name:"contract" help:"contract address" required:"true"`
-	Collection       string              `arg:"" name:"collection" help:"collection symbol" required:"true"`
+	Collection       string              `arg:"" name:"collection" help:"collection name" required:"true"`
```

Comment:
collection-id(contract-id) is different with collection name(just string).
collection-symbol in old mitum-nft must be an instance of contract-id.
It seems that `collection` in mint operation is collection-id, not collection name.
So it would be better to replace `help:"collection name"` with `help:"collection id"(or just "collection")`.

Reply:
> collection-id(contract-id) is different with collection name(just string).
> collection-symbol in old mitum-nft must be an instance of contract-id.
> It seems that `collection` in mint operation is collection-id, not collection name.
> So it would be better to replace `help:"collection name"` with `help:"collection id"(or just "collection")`.

Ok I will fix it.

Reply:
@wyuinche Done!